nautobot/docs/release-notes/version-2.0.md

```diff
@@ -14,4 +14,149 @@ If you are a user migrating from Nautobot v1.X, please refer to the ["Upgrading
+|                                | power_panel_id     | Deleted                                                                      |
+|                                | rack_id            | Deleted                                                                      |
+
+There are also instances where a foreign-key related field is incorrectly mapped to a boolean membership filter.
+
+For example:
+
+```python
+    has_power_outlets = RelatedMembershipBooleanFilter(
+            field_name="poweroutlets",
+            label="Has power outlets",
+        )
+    power_outlets = has_power_outlets
+```
+
+Now we change the foregin-key related field to be its own PK based filter:
+
+```python
+    has_power_outlets = RelatedMembershipBooleanFilter(
+            field_name="poweroutlets",
+            label="Has power outlets",
+        )
+    power_outlets = django_filters.ModelMultipleChoiceFilter(
+        queryset=PowerOutlet.objects.all(),
+        field_name="poweroutlets",
+        label="Power Outlets",
+    )
+```
+
+Below is a table documenting all such changes and where they occurred.
+
+| Filterset       | Old Filter Field     | Changes                                                                           |
+|-----------------|----------------------|-----------------------------------------------------------------------------------|
+| DeviceFilterSet | console_ports        | Changed from mapping to `has_console_ports` to `ModelMultipleChoiceFilter`        |
+|                 | console_server_ports | Changed from mapping to `has_console_server_ports` to `ModelMultipleChoiceFilter` |
+|                 | power_ports          | Changed from mapping to `has_power_ports` to `ModelMultipleChoiceFilter`          |
+|                 | power_outlets        | Changed from mapping to `has_power_outlets` to `ModelMultipleChoiceFilter`        |
+|                 | interfaces           | Changed from mapping to `has_interfaces` to `ModelMultipleChoiceFilter`           |
+|                 | front_ports          | Changed from mapping to `has_front_ports` to `ModelMultipleChoiceFilter`          |
+|                 | rear_ports           | Changed from mapping to `has_rear_ports` to `ModelMultipleChoiceFilter`           |
+|                 | device_bays          | Changed from mapping to `has_device_bays` to `ModelMultipleChoiceFilter`          |
+|                 | pass_through_ports   | Deleted in exchange for in exchange for `has_(front/rear)_ports`                  |
+
+!!! important
+    `NaturalKeyOrPKMultipleChoiceFilter` is not filtering on true Natural Keys in the case where its `to_field_name` argument is mapped to the `name` attribute or a single field that is not enough to identify an unique model instance by itself. This is technically a bug and is tracked here <https://github.com/nautobot/nautobot/issues/2875> and should be resolved after concluding this Epic <https://github.com/nautobot/nautobot/issues/1574>.
```

Comment:
A concrete example here might be more helpful to the end user than the technical details of how it's implemented.

Reply:
I feel like this portion might be too technical for an end user facing documentation so I am going to delete this. 
We also have the #TODO comments in this [PR](https://github.com/nautobot/nautobot/pull/2882/) to track this issue internally.